agent/src/agent/pipeline/manager.py

```diff
@@ -80,12 +80,15 @@ def check_pipeline_id(pipeline_id: str):
 
 def start(pipeline_: Pipeline, wait_for_sending_data: bool = False):
     reset_pipeline_retries(pipeline_)
+    set_pipeline_retries_notification_sent(pipeline_, False)
     sdc_client.start(pipeline_, wait_for_sending_data)
 
 
 def stop(pipeline_: Pipeline):
     try:
         sdc_client.stop(pipeline_)
+        reset_pipeline_retries(pipeline_)
+        set_pipeline_retries_notification_sent(pipeline_, False)
```

Comment:
I think if we reset pipeline retries, we should always reset notification as well, if so, you can put this call inside `reset_pipeline_retries`